tt_metal/hw/inc/dev_msgs.h

```diff
@@ -112,9 +112,19 @@ static_assert(MEM_MAILBOX_BASE + offsetof(mailboxes_t, ncrisc_halt.stack_save) =
 static_assert(MEM_MAILBOX_BASE + sizeof(mailboxes_t) < MEM_MAILBOX_END);
 #endif
 
+enum EthRouterMode : uint32_t {
+    FD_SRC = 0,
+    FD_DST = 1,
+    SD = 2,
+};
+
 struct routing_info_t {
     volatile uint32_t routing_enabled;
     volatile uint32_t routing_mode;
-    volatile uint32_t unused_arg0;
-    volatile uint32_t unused_arg1;
+    volatile uint32_t connected_chip_id;
+    volatile uint32_t dispatch_core_xy;
+    volatile uint32_t fd_buffer_msgs_sent;
+    uint32_t reserved_0_;
+    uint32_t reserved_1_;
+    uint32_t reserved_2_;
```

Comment:
what's the padding?  why 6 uint32?
I think we should pack this tight, so use the smallest applicable datatype.  in refererence to the XY noc address comment above, packing those into a 16 byte value and accessing with 8 bit mask/shfit would be efficient (and save an L1 load).

Reply:
`fd_buffer_msgs_sent` is getting sent with ethernet hardware which acts on 16 byte chunks. The receiver eth core writes to this address on the sender eth core. I agree about the 8 bit mask/shift for perf optimization